src/js/background_scripts/message-listener-utils.js

```diff
@@ -139,4 +139,30 @@ https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
             }
         });
     };
+
+    /**
+     *
+     * Change the zoom value of the current browser tab.
+     *
+     * @param {String} zoomType - Determines if the page should be zoomed in or out.
+     *
+     */
+    gamepad.messageListenerUtils.setZoom = function (zoomType) {
+        chrome.tabs.query({ currentWindow: true, active: true }, function (currentTab) {
+            // Obtain the zoom value of the current tab.
+            chrome.tabs.getZoom(currentTab.id, function (currentZoomFactor) {
+                // Compute the new zoom value according to the zoom type.
+                var newZoomFactor = null;
+                if (zoomType === "zoomIn") {
+                    newZoomFactor = currentZoomFactor + 0.1;
```

Comment:
Was there some reason you didn't add a speed factor to the control panel and/or make this a configurable option?

Reply:
@the-t-in-rtf I didn't think of adding `speedFactor` while writing this because usually, users won't need anything to decide with what speed they want to zoom in (unlike the case we do have in scrolling and tab navigation where users might want to set up the navigation speed). I can add it if you think that'd be better.

Reply:
I think the increments are reasonable and fairly close to what the keyboard delivers.  Most of my concerns are with the extreme zooms generated by slowly depressing the triggers, and we can address that in #22.